waypoints/northern-california-sprint-league/northern-california-sprint-league.wpt

```diff
@@ -1,46 +1,46 @@
-BEGIN SYMBOL
```

Comment:
Ah cool. Thanks yeah I don't have fly sky so I didn't know these were not ideal for that. I will ask for feedback from iOS users in the future :D